framework/src/application/node/node.js

```diff
@@ -318,14 +318,18 @@ module.exports = class Node {
 			}),
 			getLastBlock: async () => this.processor.serialize(this.chain.lastBlock),
 			getBlocksFromId: async action =>
-				this.transport.handleRPCGetBlocksFromId(
-					action.params.data,
-					action.params.peerId,
+				this.processor.serialize(
+					this.transport.handleRPCGetBlocksFromId(
+						action.params.data,
+						action.params.peerId,
+					),
 				),
 			getHighestCommonBlock: async action =>
-				this.transport.handleRPCGetGetHighestCommonBlock(
-					action.params.data,
-					action.params.peerId,
+				this.processor.serialize(
+					this.transport.handleRPCGetGetHighestCommonBlock(
```

Comment:
for this one it dint have.. so maybe just follow the same pattern add the `serialize` in `handleRPCGetGetHighestCommonBlock` function? framework/src/application/node/transport/transport.js